testing/spectest/general/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ go_test(
     srcs = ["deneb__kzg__verify_blob_kzg_proof_batch_test.go"],
     data = ["@consensus_spec_tests//:test_data"],
     tags = ["spectest"],
+    timeout = "short",
```

Comment:
how does setting timeout to short help with time out? if i understand the PR description and issue correctly

Reply:
Bazel is complaining that we have a moderate timeout for a short test. So this timeout of short would resolve the warning from bazel.